PendingReleaseNotes.md

```diff
@@ -15,3 +15,4 @@ Object:
 ## Features
 
 - Support external mons for local Rook cluster (see [#14733](https://github.com/rook/rook/issues/14733)).
+- add credentials/keypair management to CephObjectStoreUser (see [#15359](https://github.com/rook/rook/issues/15359)
```

Comment:
Thanks for adding this to pending notes. I think it would be best to put this in the breaking changes section, with the same caution you mentioned in the description.

> Note that this will cause manually created credentials/keypairs on an rgw user to be purged. This behavior is necessary to ensure that keypairs that were explicitly declared and then undeclared are purged. This could be a user observable regression. Out of an abundance of caution, it is recommend that this feature not be introduce into a stable minor release series.

Reply:
Done.